### ISSUE TITLE
Stream name filtering broken?

### ISSUE THREAD

**Issue by bedge** [jorgebastida/awslogs#158]
I get this any time I try to use a stream name:
`No streams match your pattern 'syslog' for the given time period`

Usine ALL works:
`awslogs get  my_group ALL`

Using a stream name does not:
`awslogs get  my_group my_stream`

Whatever I use in place of `my_stream`, always outputs:
`No streams match your pattern 'ml_stream' for the given time period`

despite `my_stream` being present in `awslogs streams my_group`

e.g.:
```
%> awslogs get  dc-15-ur-api nn-api-egress.log
No streams match your pattern 'nn-api-egress.log' for the given time period.

%> awslogs streams dc-15-ur-api
nn-api-egress.log
ur-api.log

%> awslogs get  dc-15-ur-api ALL
dc-15-ur-api nn-api-egress.log Oct 16 13:25:33 ip-10-16-57-76 [6036] nimnn.manager(shlock:257) - DEBUG: context sh lock exit
dc-15-ur-api nn-api-egress.log Oct 16 13:25:33 ip-10-16-57-76 [6036] nimnn.manager(lock_set:43) - DEBUG: Release lock: 10.16.57.76
dc-15-ur-api nn-api-egress.log Oct 16 13:25:33 ip-10-16-57-76 [5951] nimnn.manager(shlock:253) - DEBUG: context sh lock enter
dc-15-ur-api nn-api-egress.log Oct 16 13:25:33 ip-10-16-57-76 [5951] nimnn.manager(lock_set:46) - DEBUG: Wait for SH 10.16.57.76
```

```
%> brew info awslogs
awslogs: stable 0.10 (bottled), HEAD
Simple command-line tool to read AWS CloudWatch logs
https://github.com/jorgebastida/awslogs
/usr/local/Cellar/awslogs/0.10 (1,270 files, 30.2MB) *
  Poured from bottle on 2017-10-16 at 13:15:40
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/awslogs.rb
```

**Comment by ccann** [comment]
Seeing the same thing

**Comment by kurtwheeler** [comment]
I am also experiencing this issue. I get the following output if I try to use stream matching:
```
$ awslogs get data-refinery-log-group-kurtwheeler-dev log-stream-foreman-kurtwheeler-dev --start='1 hours'
No streams match your pattern 'log-stream-foreman-kurtwheeler-dev' for the given time period.
```
However if I instead use grep to filter strings I get output:
```
$ awslogs get data-refinery-log-group-kurtwheeler-dev ALL --start='1 hours' | grep log-stream-foreman-kurtwheeler-dev
data-refinery-log-group-kurtwheeler-dev log-stream-foreman-kurtwheeler-dev 2018-07-11 18:03:26,284  <REDACTED>
...
```

I'll note that I ran those two commands one after another and it seems to be consistent.

**Comment by jorgebastida** [comment]
https://github.com/jorgebastida/awslogs/pull/199 should fix this issue. These changes will be released as part of ``0.11.0``.
